### ISSUE TITLE
Uuid support stopped working

### ISSUE THREAD

**Issue by krojew** [AlexPikalov/cdrs#301]
In version 2.2.2 converting to/from uuid results in an error:

```
the trait `cdrs::types::IntoRustByName<uuid::Uuid>` is not implemented for `cdrs::types::rows::Row`
the trait `std::convert::From<uuid::Uuid>` is not implemented for `cdrs::types::value::Bytes`
the trait `cdrs::types::IntoRustByIndex<uuid::Uuid>` is not implemented for `cdrs::types::rows::Row
````

**Comment by AlexPikalov** [comment]
@krojew I'm on it

**Comment by AlexPikalov** [comment]
@krojew It may happen if in your application you use UUID version different from `0.8.1`.

**Comment by jacekchmiel** [comment]
@AlexPikalov did you consider changing `uuid = "0"` requirement to `uuid = "0.8.1"` if it actually requires this version specifically?

**Comment by AlexPikalov** [comment]
@jacekchmiel yup, fixing version (which I'm not a big fan of) must be only the only solution in this case.

**Comment by AlexPikalov** [comment]
I've just published CDRS version `2.2.4` which contains fixed version of UUID. It must fix the problem of version inconsistency/multiple version. Also it must fix possible issues with non-semantic versioning of UUID when 0.7 and 0.8 despite having the same major version is not compatible and 0.8 contains a breaking change comparing to 0.7.
